src/Compilers/Core/Portable/Serialization/StreamObjectReader.cs

```diff
@@ -1043,53 +1049,39 @@ private object ReadBoxedEnum()
             throw ExceptionUtilities.UnexpectedValue(enumType);
         }
 
-        private Variant ReadObject(EncodingKind kind)
+        private Variant ReadObject()
         {
-            switch (kind)
-            {
-                case EncodingKind.ObjectRef_4Bytes:
```

Comment:
these cases moved up to the caller.  the rest of the method was just dedented but did nother otherwise really change.